firebase-database/src/androidTest/java/com/google/firebase/database/core/persistence/KeepSyncedTest.java

```diff
@@ -57,12 +57,10 @@ private void assertIsKeptSynced(Query query) throws Exception {
 
     new ReadFuture(
             query,
-            new ReadFuture.CompletionCondition() {
-              public boolean isComplete(List<EventRecord> events) {
```

Comment:
`isComplete` needs an `@Override` annotation or we could alternatively just use lambda syntax.